### PR TITLE
Roadmap: 2023 Update

### PR DESCRIPTION
**Pull request by ceb8**
### Purpose:

Updating the Roadmap document per discussion at the Astropy Coordination meeting.

Notes from the Roadmap session are [here](https://docs.google.com/document/d/1sBImbw1p4mM3VYWGQJDRbj2N3uqP5Ed_Wfb8BDjJj2o/edit#heading=h.5mbylhiv4ljb).

### Feedback:

- This PR is open for comment until 31 May 2023
- Consider what on here has been solved and what issues are missing in your area of specialty

### CODE REVIEW

File: roadmap/roadmap.md [hunk 1]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
-## Roadmap Meta Issues
 
-Meta issues in this repository associated with Roadmap items can be found [here](https://github.com/astropy/astropy-project/issues?q=is%3Aopen+is%3Aissue+label%3ARoadmap). Work done anywhere in the project that is relevant to the Roadmap can be tied to the appropriate meta issue for visibility.
 
 ## Functionality
 
 - :large_orange_diamond: Provide next-generation spectroscopic reduction, analysis, and visualization tools usable by individual researchers and larger surveys.
 
-- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy including FITS and HDF5 as well as compression handling and enhanced support for large tables.
+- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified i/o system.
```
Comment (pllim) [inline roadmap/roadmap.md:40]:
```suggestion
- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy by improving the unified I/O system.
```

File: roadmap/roadmap.md [hunk 2]
```diff
@@ -13,7 +13,8 @@
 
 ## Introduction
 
-This Roadmap captures high level actionable items that we as a project aim to undertake to improve the health and stability of the Astropy project. The Roadmap itself is a static document, while specific issues and efforts can be linked through [meta issues](#Roadmap-Meta-Issues). The Roadmap document will be revisited regularly at the Astropy coordination meetings, to keep track of progress and write new versions as needed.
+This Roadmap captures high level actionable items that we as a project aim to undertake to improve the health and stability of the Astropy project. The Roadmap itself is a static document, however it will be revisited regularly at the Astropy coordination meetings, to keep track of progress and write new versions as needed.
+
```
Comment (pllim) [inline roadmap/roadmap.md:17]:
```suggestion
```

File: roadmap/roadmap.md [hunk 3]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
+- :red_square: Improve support for using Astropy tools in heterogeneous computing environments such as cloud environments or GPU systems.
 
-- :red_square: Improve support for using Astropy tools in high-performance machine learning and data analysis frameworks, such as on GPU systems.
+## Learn and User Support
 
-## Learn and User Support 
+- :green_circle: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research.
 
-- :green_circle: Improve discoverability of documentation and educational materials by overhauling the Learn website, including implementing an integrated search functionality and building an example gallery.
+-  :large_orange_diamond:Update Astropy website: rework the Team page, and improve the visibility of Astropy Learn
```
Comment (pllim) [inline roadmap/roadmap.md:54]:
```suggestion
-  :large_orange_diamond: Update Astropy website: Rework the Team page and improve the visibility of Astropy Learn.
```

File: roadmap/roadmap.md [hunk 4]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
 - :large_orange_diamond: Expand and build on workshop offerings by recruiting and training more facilitators and expanding the geographic diversity of meetings where we offer workshops. 
 
-- :green_circle: Expand the number and diversity of user support resources, including establishing a user-focused forum on Discourse.
+- :large_orange_diamond: Expand the number and diversity of active user support resources.
 
-- :large_orange_diamond: Generate and ingest guides and/or a series of tutorials that demonstrate Astropy Project functionality in the context of astronomical research, especially focusing on spectroscopy tasks.
+- :red_square: Deveop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
```
Comment (pllim) [inline roadmap/roadmap.md:60]:
```suggestion
- :red_square: Develop tutorials that demonstrate the use of Astropy for spectroscopy tasks.
```

File: roadmap/roadmap.md [hunk 5]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
-- :red_square: Implement robust performance benchmark reporting for pull requests.
+- :large_orange_diamond: Improve documentation for infrastructure setup, as well as the developer documentation.
 
-- :red_square: Implement integration testing for core, coordinated, and infrastructure packages.
+- :red_square: Implement robust performance benchmark reporting.
 
 ## Government, Management, and Personnel
 
-- :large_orange_diamond: Develop a process for recruiting, selecting, and managing personnel to complete Project priorities that require external personnel.
+- :green_circle: Adopt the NumFOCUS Code of Conduct (CoC) and build a community ombuds/CoC team to curate the guidelines and do enforcement.
```
Comment (pllim) [inline roadmap/roadmap.md:82]:
```suggestion
- :green_circle: Adopt the NumFOCUS Code of Conduct (CoC) and build a community ombuds/CoC team to curate the guidelines and enforce them.
```

File: roadmap/roadmap.md [hunk 6]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Define and document the process for categorization of core, coordinated, and affiliated packages.
+- :large_orange_diamond: Develop a process for recruiting, selecting, and managing personnel to complete Project priorities that require external personnel.
```
Comment (pllim) [inline roadmap/roadmap.md:84]: Not sure if "help" is a good replacement but "personnel" seems repetitive.
```suggestion
- :large_orange_diamond: Develop a process for recruiting, selecting, and managing personnel to complete Project priorities that require external help.
```

File: roadmap/roadmap.md [hunk 7]
```diff
@@ -30,59 +31,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
-- :red_square: Improve the process for recruiting and onboarding maintainers, paying particular attention to packages that are lacking maintainers.
+- :red_square: Document the process for taking on a formal role in the Astropy project and recruit for unfilled maintainer positions.
```
Comment (pllim) [inline roadmap/roadmap.md:86]:
```suggestion
- :red_square: Document the process for taking on a formal role in the Astropy project and recruit for unfilled Team positions.
```

**Comment by pllim** [comment]
Some minor comments above. We can discuss them tomorrow. Thanks!

File: roadmap/COMPLETED.md [hunk 8]
```diff
@@ -4,6 +4,18 @@ This document is a list of former Astropy Roadmap items that have been removed b
 They are listed by the date of the revision they were first removed from.
 This document is in reverse chronological order.
 
+## May 2023
+
+### Functionality
+
+- Compression support for FITS and HDF5
```
Comment (pllim) [inline roadmap/COMPLETED.md:11]:
```suggestion
- Compression support for FITS and HDF5.
```

File: roadmap/roadmap.md [hunk 9]
```diff
@@ -30,59 +30,56 @@ This Roadmap captures high level actionable items that we as a project aim to un
...
 
 ## Community Building and Sustainability
 
 - :large_orange_diamond: Increase the learning and mentoring opportunities for people interested in becoming contributors and helping to develop existing contributors.
 
-- :large_orange_diamond: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improved our understanding the demographics of our communities in order to measure the effectiveness of these efforts.
-
 - :large_orange_diamond: Better understand Astropy user community through a NumPy-like user survey (see [this summary](https://numpy.org/user-survey-2020-details/) of their results).
 
+- :red_square: Increase inclusion, diversity, and empowerment efforts within the Astropy Project and NumFOCUS communities, and improved our understanding the demographics of our communities in order to measure the effectiveness of these efforts.
```
Comment (maxnoe) [inline roadmap/roadmap.md:69]: and improve our understanding of the

Comment (wtgee) [reply roadmap/roadmap.md:69]: "...and improve our understanding of the..."

Also need the extra "of" after understanding.

**Comment by ceb8** [comment]
The community comment period is passed and this PR is now ready for merge. (@pllim can you do this? I don't have the requiste repository access).

**Review by pllim — APPROVED** [review]
Sure thing. Thank you so much for doing this!

**Comment by pllim** [comment]
@ceb8 , do you want to make the announcement that this has been merged on astropy-dev and on Slack `#project` channel? 😸

**Pull request merged**